src/pages/item/index.js

```diff
@@ -617,23 +617,23 @@ function Item() {
                             className={`text-and-image-information-wrapper price-info-wrapper`}
                         >
                             <div className="price-wrapper price-wrapper-bright">
-                                <div>
+                                <>
                                     {t('Change vs yesterday')}
                                     {': '}
                                     {currentItemData.changeLast48h} ₽
                                     {' / '}
                                     {currentItemData.changeLast48hPercent} %
-                                </div>
-                                <div>
```

Comment:
This are too for layout
![45624D2F-C28D-4CED-8783-78F35FC19A84](https://user-images.githubusercontent.com/10927011/185232950-4392b07d-3928-40a7-b096-e246ac475342.jpeg)